app/models/doi.rb

```diff
@@ -674,82 +674,137 @@ def as_indexed_json(_options = {})
     }
   end
 
-
-  def self.query_aggregations(disable_facets: false)
-    if !disable_facets
-      {
-        # number of resourceTypeGeneral increased from 16 to 28 in schema 4.4
-        resource_types: { terms: { field: "resource_type_id_and_name", size: 30, min_doc_count: 1 } },
-        states: { terms: { field: "aasm_state", size: 3, min_doc_count: 1 } },
-        published: {
-          date_histogram: {
-            field: "publication_year",
-            interval: "year",
-            format: "year",
-            order: {
-              _key: "desc",
-            },
-            min_doc_count: 1,
+  def self.all_doi_aggregations
```

Comment:
At this point it could be a constant.
```
AGGREGATION_DEFINITIONS = {...
...
}
```

Reply:
@jrhoads Do you have thoughts on where the constant should be defined?

Reply:
It can be a constant within the model.